ppo.ipynb

```diff
@@ -0,0 +1,352 @@
+   "source": [
+    "# Configuration\n",
+    "ENV_NAME = \"Pendulum-v1\""
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "y7uMVaYaFcA0"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "# Create an environment, grab the spec\n",
+    "# We should use our own environment here\n",
+    "environment = helpers.make_environment(ENV_NAME)\n",
+    "environment_spec = specs.make_environment_spec(environment)\n",
+    "\n",
+    "# environment = PendulumEnv()\n",
+    "# environment_spec = specs.make_environment_spec(environment)"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "rltCiwH2FYWA"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "# Create the networks\n",
+    "# We should use our own netowkrs here\n",
+    "# agent_networks = dm_ppo.make_gym_networks(environment_spec)"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "kfJrCivxHmqH"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "# Create the agent\n",
+    "# We should use PPO agent\n",
+    "agent = RandomAgent(environment_spec)"
+   ],
+   "outputs": [],
+   "metadata": {
+    "id": "xpJvcAOtF2t3"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Interaction loop"
+   ],
+   "metadata": {
+    "id": "kluuB3xPJd_u"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "source": [
+    "def training_loop(\n",
+    "    environment,\n",
+    "    agent,\n",
+    "    num_episodes=None,\n",
+    "    num_steps=None,\n",
+    "    logger_time_delta=1.0,\n",
+    "    label=\"training_loop\",\n",
+    "):\n",
+    "    \"\"\"Perform the run loop.\n",
+    "\n",
+    "    We are following the Acme run loop.\n",
+    "\n",
+    "    Run the environment loop for `num_episodes` episodes. Each episode is itself\n",
+    "    a loop which interacts first with the environment to get an observation and\n",
+    "    then give that observation to the agent in order to retrieve an action. Upon\n",
+    "    termination of an episode a new episode will be started. If the number of\n",
+    "    episodes is not given then this will interact with the environment\n",
+    "    infinitely.\n",
+    "\n",
+    "    Args:\n",
+    "      environment: dm_env used to generate trajectories.\n",
+    "      agent: acme.Actor for selecting actions in the run loop.\n",
+    "      num_steps: number of episodes to run the loop for. If `None` (default), runs\n",
+    "        without limit.\n",
+    "      num_episodes: number of episodes to run the loop for. If `None` (default),\n",
+    "        runs without limit.\n",
+    "      logger_time_delta: time interval (in seconds) between consecutive logging\n",
+    "        steps.\n",
+    "      label: optional label used at logging steps.\n",
+    "    \"\"\"\n",
+    "    logger = loggers.TerminalLogger(label=label, time_delta=logger_time_delta)\n",
```

Comment:
I don't think we really need a logger here since the use of a logger is to print in the terminal but here we are using a jupyter notebook... 
Maybe I missed something, just tell me :)